framework/test/gtest/ImportTestBundles.cpp

```diff
@@ -0,0 +1,23 @@
+#include "cppmicroservices/BundleActivator.h"
```

Comment:
This looks to be functionally equivalent to ```framework/test/driver/TestDriverActivator.cpp```. Why not have one source files shared by the two test frameworks?